tap_listrak/__init__.py

```diff
@@ -22,6 +22,7 @@ def discover(ctx):
     for tap_stream_id in schemas.stream_ids:
         schema = Schema.from_dict(schemas.load_schema(tap_stream_id),
                                   inclusion="automatic")
+        schema.selected = True if tap_stream_id in ['lists', 'messages'] else False
```

Comment:
why add this line?

Reply:
So lists and messages should be on by default, per Buck's README, right? So this turns them on and then everything else is false unless turned on. It took me a little while to realize that was what was breaking it when I was running locally. 

Before selected wasn't there at in the catalog.

Reply:
oh, I understand -- this is in discovery. this makes sense.